internal/provisioner/kops_provisioner.go

```diff
@@ -298,14 +300,21 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, aws aw
 	}
 
 	// Get the new ELB internal-nginx endpoint
-	endpoint, err := getLoadBalancerEndpoint("internal-nginx", logger, kops.GetKubeConfigPath())
+	logger.Infof("Waiting up to %d seconds for internal ELB to be ready...", wait)
+	defer cancel()
```

Comment:
This would cancel a context created above which has already been counting down for some amount of time. A new context should be made here.

Reply:
New context created